api/accesslog.proto

```diff
@@ -169,3 +169,30 @@ message EnvoyAccessLog {
   google.protobuf.UInt32Value tls_cipher_suite = 30;
 }
 
+// Stream message for the StreamAccessLogs API. Envoy will open a stream to the server and stream
+// access logs without ever expecting a response.
+message EnvoyAccessLogMessage {
+  // The node sending the access log messages over the stream. Note that the server should only
+  // expect node to be populated on the first message in the stream. It will not be populated in
+  // subsequent messages. This is a performance optimization.
+  Node node = 1;
+  // The individual access log message.
+  EnvoyAccessLog log = 2;
+}
+
+// Empty response for the StreamAccessLogs API. Will never be sent. See below.
+message EnvoyAccessLogResponse {}
+
+// Service for streaming access logs from Envoy to an access log server.
+service EnvoyAccessLogService {
+  // Envoy will connect and send EnvoyAccessLogMessage messages forever. It does not expect any
+  // response to be sent as nothing would be done in the case of failure. The server should
+  // disconnect if it expects Envoy to reconnect. In the future we may decide to add a different
+  // API for "critical" access logs in which Envoy will buffer access logs for some period of time
+  // until it gets an ACK so it could then retry. This API is designed for high throughput with the
+  // expectation that it might be lossy.
+  rpc StreamAccessLogs(stream EnvoyAccessLogMessage)
```

Comment:
How does the server get configured in the bootstrap for this?

Reply:
This will be a new well known name for a log handler. I will update the docs, etc. to make it more clear.